NC-2740: Update Turnstile Documentation to reflect some callback changes (#8517)
cloudflare/cloudflare-docs: content/turnstile/frequently-asked-questions.md

You can find special sitekeys to be used for testing in the [testing](/turnstile/reference/testing/) section.

{{</faq-answer>}}
{{</faq-item>}}

{{<faq-item>}}

> {{<faq-question level=2 text="Is there an API to create or managed Turnstile Widgets?" >}}

Reviewer: ```suggestion
{{<faq-question level=2 text="Is there an API to create or manage Turnstile widgets?" >}}
```

{{<faq-answer>}}

> Yes, you can find the API for managing Turnstile widgets [here](https://developers.cloudflare.com/api/operations/accounts-turnstile-widgets-list).

Reviewer: ```suggestion
Yes, you can refer to the [Turnstile API documentation](/api/operations/accounts-turnstile-widgets-list) to manage your widgets.
```

{{</faq-answer>}}
{{</faq-item>}}

{{<faq-item>}}
{{<faq-question level=2 text="I am seeing a 401 error in your console during a Turnstile security check, is this a problem?" >}}
{{<faq-answer>}}

> You can safely ignore the error, it's requesting a PAT but your device/browser does not support PATs yet, so you see the error.

Reviewer: ```suggestion
You can safely ignore the error. It is requesting a PAT that your device or browser does not support yet.
```
